README.md

```diff
@@ -36,7 +36,7 @@ npm i -g zx
 ## Goods
 
 [$](#command-) · [cd()](#cd) · [fetch()](#fetch) · [question()](#question) · [sleep()](#sleep) · [echo()](#echo) · [stdin()](#stdin) · [within()](#within) ·
-[chalk](#chalk-package) · [fs](#fs-package) · [os](#os-package) · [path](#path-package) · [glob](#globby-package) · [yaml](#yaml-package) · [minimist](#minimist-package) · [which](#which-package) ·
+[chalk](#chalk-package) · [fs](#fs-package) · [opn](#open-package) · [os](#os-package) · [path](#path-package) · [glob](#globby-package) · [yaml](#yaml-package) · [minimist](#minimist-package) · [which](#which-package) ·
```

Comment:
Is there a reason why it's called `opn` instead of `open`? I think the later is clearer.

Reply:
Some weird compilation error. It looks like the open keyword is reserved, so I'm not able to use it.

Reply:
You are right, I tried it and had this error
```
Duplicate identifier 'open'.ts(2300)
lib.dom.d.ts(17754, 18): 'open' was also declared here.
```
It seems that Typescript considers the global object to have properties of `window` in the browser, which has an [`open` method](https://developer.mozilla.org/en-US/docs/Web/API/Window/open).

This also explains why `fetch` is not declared in the `globals.ts` file, because it's already part of `window` (and was also added to `node` global context).

I don't see the "DOM" lib on `tsconfig.json` so I am not sure where is it coming from :\

Reply:
From node-fetch =(

Reply:
Ah, I see. Do we really need `node-fetch` now that `node` has `fetch` by default?

Reply:
> node has fetch by default

We can! But right now zx support >= node16. So until the next major zx we can't change it.